src/dashboard/Data/Browser/Browser.react.js

```diff
@@ -563,7 +621,8 @@ class Browser extends DashboardView {
     }
     obj.save(null, { useMasterKey }).then(
       objectSaved => {
-        const msg = objectSaved.className + ' with id \'' + objectSaved.id + '\' created';
+        // eslint-disable-next-line quotes
+        const msg = objectSaved.className + " with id '" + objectSaved.id + "' created";
```

Comment:
Correct me if I'm wrong, but I think this can be reverted. Same on line 728. And few other places.

Reply:
lint fails if i dont do that 
@404-html

Reply:
yeah it was caused by some mistake my bad